project-base/app/tests/FrontendApiBundle/Functional/Customer/User/RegisterTest.php

```diff
@@ -43,6 +43,17 @@ public function testUniqueBillingAddressIsNotValidatedInB2cRegistration(): void
         $this->assertSuccessfulRegistration($response);
     }
 
+    public function testCompanyUserRegistration(): void
+    {
+        $registerQueryVariables = self::getRegisterQueryVariables();
+        $registerQueryVariables['companyCustomer'] = true;
+        $registerQueryVariables['companyNumber'] = '999';
```

Comment:
this company number will pass the validation? I thought it has to be 8 digits

Reply:
seems like there is no such validation on the API side (IMHO there should not be as this is country-specific)

https://github.com/shopsys/shopsys/blob/16.0/packages/frontend-api/src/Resources/config/graphql-types/ModelType/Customer/BillingAddress/Input/CompanyInputObjectDecorator.types.yaml#L23